web/themes/joinup/joinup_theme.theme

```diff
@@ -43,6 +54,34 @@ function joinup_theme_preprocess_page(&$variables) {
     if ($entity_type == 'collection' || $entity_type == 'custom_page') {
       $variables['is_collection'] = TRUE;
     }
+
+    if ($entity_type == 'collection' && isset($variables['page']['content']['pagetitle'])) {
+      $title = $variables['page']['content']['pagetitle'];
+      unset($variables['page']['content']['pagetitle']);
+    }
+  }
+
+  $variables['title'] = $title;
+}
+
+/**
+ * Implements hook_preprocess_page_title().
+ */
+function joinup_theme_preprocess_page_title(&$variables) {
+  $route_match = \Drupal::routeMatch();
+  $route_name = $route_match->getRouteName();
+
+  // Array contains create pages.
```

Comment:
Here also improve the documentation. Something explaining the reason of what we are doing.
`// Set a flag when we are on content creation routes in order to customise some templates.`